src/App.css

```diff
@@ -1,12 +1,12 @@
-@import url("https://rsms.me/inter/inter.css");
```

Comment:
the transparency hadn't been imported correctly so I had to change that to make them work properly.